easybuild/easyconfigs/h/h5py/h5py-2.7.1-foss-2017b-Python-3.6.3.eb

```diff
@@ -16,15 +16,19 @@ source_urls = [PYPI_SOURCE]
 sources = [SOURCE_TAR_GZ]
 checksums = ['180a688311e826ff6ae6d3bda9b5c292b90b28787525ddfcb10a29d5ddcae2cc']
 
-# to really use mpi enabled hdf5 we now seem to need a configure step
-prebuildopts = ' python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
+# To really use mpi enabled hdf5 we now seem to need a configure step
+# Works with pip. Tested with examples in http://docs.h5py.org/en/stable/mpi.html
+preinstallopts = ' python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
```

Comment:
@vanzod Please move this down below the `dependencies` block while you're at it?